form/data_transformers.rst

```diff
@@ -342,7 +342,7 @@ First, create the custom field type class::
     namespace AppBundle\Form;
 
     use AppBundle\Form\DataTransformer\IssueToNumberTransformer;
-    use Doctrine\Common\Persistence\ObjectManager;
+    use Doctrine\Persistence\ManagerRegistry;
```

Comment:
This is not used in this example as far as I can see, let's remove the use statement?